template/libtofs.jinja

```diff
@@ -5,7 +5,7 @@
                        v1_path_prefix='') %}
   {#-
     Returns a valid value for the "source" parameter of a "file.managed"
-    state function. This makes easier the usage of the Template Override and
+    state function. This makes easier the usage of the template Override and
```

Comment:
Not this one, this is `Template` as in `TOFS` (file templates, not the `template-formula`)!